lib/rules/lines-around-comment.js

```diff
@@ -93,6 +93,12 @@ module.exports = {
                     },
                     allowArrayEnd: {
                         type: "boolean"
+                    },
+                    ignorePattern: {
+                        type: "string"
+                    },
+                    applyDefaultPatterns: {
```

Comment:
@not-an-aardvark I already used the new name in this rule. Only the existing option in `line-comment-position` will be deprecated.